libethcore/Farm.h

```diff
@@ -117,8 +117,6 @@ class Farm: public FarmFace
 
 		// Set work to each miner
 		Guard l(x_minerWork);
-		if (_wp.header == m_work.header && _wp.startNonce == m_work.startNonce)
```

Comment:
This is no longer necessary since we'll never call this function with these conditions.